Tests/SwiftTryCatchTests/SwiftTryCatchTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  SwiftTryCatch.h
+//
+//  Created by Dan Federman on 12/20/19.
+//  Copyright (c) 2019 Dan Federman.
```

Comment:
Happy to give you copyright if you'd like. I intentionally left off "All rights reserved", since the MIT permission comment-block below conflicts with the "All rights reserved" line.